[Wrangler/D1] Document Initial d1 commands in the right place (#7202)
cloudflare/cloudflare-docs: content/workers/wrangler/commands.md

---

## d1

Interact with Cloudflare's D1 service.

{{<Aside type="note">}}

> D1 is currently in open alpha and is not recommended for production data and traffic.
> 
> Please report any bugs to https://github.com/cloudflare/wrangler2/issues/new/choose.

Reviewer: ```suggestion
D1 is currently in open alpha and is not recommended for production data and traffic. Report D1 bugs to the [Wrangler team](https://github.com/cloudflare/wrangler2/issues/new/choose).
```

```sh
$ wrangler d1 create <DATABASE_NAME>
```

{{<definitions>}}

- `DATABASE_NAME` {{<type>}}string{{</type>}} {{<prop-meta>}}required{{</prop-meta>}}
  - The name of the new D1 database.

{{</definitions>}}

### `list`

List all D1 databases in your account.

```sh
$ wrangler d1 list
```

### `delete`

Delete a D1 database.

```sh
$ wrangler d1 delete <DATABASE_NAME>
```

{{<definitions>}}

- `DATABASE_NAME` {{<type>}}string{{</type>}} {{<prop-meta>}}required{{</prop-meta>}}
  - The name of the D1 database to delete.

{{</definitions>}}

### `execute`

Execute a query on a D1 database.

```sh
$ wrangler d1 execute <DATABASE_NAME> [OPTIONS]
```

{{<definitions>}}

- `DATABASE_NAME` {{<type>}}string{{</type>}} {{<prop-meta>}}required{{</prop-meta>}}
  - The name of the D1 database to execute a query on.
- `--command` {{<type>}}string{{</type>}} 
  - The SQL query you wish to execute.
- `--file` {{<type>}}string{{</type>}} 
  - Path to the SQL file you wish to execute.
- Note that you must provide either `--command` or `--file` for this command to run successfully.
{{</definitions>}}

### `backup create`

> Initiate a D1 backup

Reviewer: ```suggestion
Initiate a D1 backup.
```
